erpnext/regional/united_arab_emirates/utils.py

```diff
@@ -30,6 +47,9 @@ def update_itemised_tax_data(doc):
 				tax_amount += flt((row.net_amount * _tax_rate) / 100, row.precision("tax_amount"))
 				tax_rate += _tax_rate
 
+		if not tax_rate and is_export:
+			row.is_zero_rated = 1
```

Comment:
```suggestion
		if not tax_rate:
			row.is_zero_rated = is_export
```

This will help you revert the condition if it's already checked.

Reply:
done